base/collection_xattr_test.go

```diff
@@ -787,13 +783,71 @@ func TestWriteTombstoneWithXattrs(t *testing.T) {
 					"_xattr1": []byte(`{"c": "d"}`),
 					"_xattr2": []byte(`{"f": "g"}`),
 				},
-				cas:        fakeCas,
-				deleteBody: false,
-				// this seems wrong, no error??
-				finalBody: []byte("{}"),
+				cas:            fakeCas,
+				deleteBody:     false,
+				writeErrorFunc: RequireDocNotFoundError,
+			},
+			{
+				name:        "previousDoc=nil,xattrsToUpdate=_xattr1,xattrsToDelete=_xattr2,cas=0,deleteBody=false",
```

Comment:
We don't need to make this change in this PR, but I think these tests would be more readable and concise with the following changes:
- define an enum for the set of possible values for 'previousDoc', 'updatedXattrs' and 'finalXattrs' (e.g. for previousDoc, "none, "bodyOnly", "bodyWithXattr1", ...) 
- use cas=incorrect in the naming, instead of cas=1 or fakeCas

Reply:
I like the idea of the enum but it becomes pretty confusing quickly because the contents of xattr1 could change values. I'm going to leave it out of this PR.